pkg/controller/kubelet-config/kubelet_config_controller.go

```diff
@@ -59,6 +60,9 @@ var updateBackoff = wait.Backoff{
 	Jitter:   1.0,
 }
 
+// ErrCouldNotFindMCPSet Error if no MCPs could be found
+var ErrCouldNotFindMCPSet = errors.New("could not find any MachineConfigPool set for KubeletConfig")
```

Comment:
does this need to be exported?

Reply:
it doesn't.... I have removed the export in the force push.